README.md

```diff
@@ -74,6 +74,10 @@ To use YLTableView, you should subclass `YLTableViewDataSource`. After implement
 @end
 ```
 
+#### Reuse identifiers and Swift
+YLTableView relies on using a cell class name as its reuse identifier. To achieve this, you should use `NSStringFromClass([YLExampleTextCell class])` even with Swift code. Using `String(YLExampleTextCell)` can cause issues with YLTableView.
```

Comment:
nit: "…as that cell's reuse identifer.…"

Also, can you meantion that you can also make ObjC use a sane name by marking your UITableViewCell's class as `@objc(MyCellSubclass)` (that just re-declares the same name)?